_source/user-guide/kibana/index.md

```diff
@@ -8,24 +8,25 @@ tags:
   - kibana
 contributors:
   - imnotashrimp
+  - shalper
```

Comment:
There are pieces in Kibana that do not receive attention and should
- The time picker
- List of fields and how is it used to quickly filter on values. The meaning of "available fields" with the notion of the Kibana index and what does it mean that a field is not there (cannot do visualizations). Fields that are unmapped (they have a question mark on the left) and the need to "refresh mapping"
- The actions menu
- Mention the Kibana advisor and how is it helping understand your search issues
- The icons menu on the left 
- The concept of filters